src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs

```diff
@@ -133,6 +133,8 @@ public ExportResult Export(in Batch<LogRecord> batch)
             }
         }
 
+        ExporterEventSource.Log.ExportSuccess(nameof(MsgPackLogExporter));
```

Comment:
@vishweshbankwar Do you have the perf number? I want to understand the cost of these.

Reply:
**Trace**
**Without Listener (With Change)**
| Method         | Mean     | Error   | StdDev  | Allocated |
|--------------- |---------:|--------:|--------:|----------:|
| ExportActivity | 148.6 ns | 0.94 ns | 0.88 ns |         - |

**Without Listener (Main)**
| Method         | Mean     | Error   | StdDev  | Allocated |
|--------------- |---------:|--------:|--------:|----------:|
| ExportActivity | 148.9 ns | 1.23 ns | 1.03 ns |         - |

**With Listener (With Change)**
| Method         | Mean     | Error   | StdDev  | Allocated |
|--------------- |---------:|--------:|--------:|----------:|
| ExportActivity | 381.3 ns | 4.03 ns | 3.77 ns |         - |

**With Listener (Main)**
| Method         | Mean     | Error   | StdDev  | Allocated |
|--------------- |---------:|--------:|--------:|----------:|
| ExportActivity | 148.2 ns | 1.01 ns | 0.90 ns |         - |


**Log**

**Without listener (With Change)**
| Method | IncludeFormattedMessage | Mean     | Error   | StdDev  | Allocated |
|------- |------------------------ |---------:|--------:|--------:|----------:|
| Export | False                   | 217.6 ns | 0.94 ns | 0.84 ns |         - |
| Export | True                    | 217.3 ns | 0.91 ns | 0.86 ns |         - |

**Without listener (Main)**
| Method | IncludeFormattedMessage | Mean     | Error   | StdDev  | Allocated |
|------- |------------------------ |---------:|--------:|--------:|----------:|
| Export | False                   | 220.3 ns | 1.67 ns | 1.48 ns |         - |
| Export | True                    | 218.7 ns | 0.65 ns | 0.54 ns |         - |


**With Listener (With Change)**
| Method | IncludeFormattedMessage | Mean     | Error   | StdDev  | Allocated |
|------- |------------------------ |---------:|--------:|--------:|----------:|
| Export | False                   | 451.2 ns | 2.91 ns | 2.72 ns |         - |
| Export | True                    | 442.1 ns | 2.13 ns | 1.99 ns |         - |


**With Listener (Main)**
| Method | IncludeFormattedMessage | Mean     | Error   | StdDev  | Allocated |
|------- |------------------------ |---------:|--------:|--------:|----------:|
| Export | False                   | 219.7 ns | 0.69 ns | 0.65 ns |         - |
| Export | True                    | 218.2 ns | 2.11 ns | 1.98 ns |         - |

Reply:
No noticeable difference when running without listener. With listener it increases as expected

Reply:
Thanks!